### PR TITLE
change actel section to work with OTN and Act data

### PR DESCRIPTION
**Pull request by jdpye**
updating the ACTEL parts of the script and adding new data sets to include receiver serials

### CODE REVIEW

File: code/day2/4_act_data_in_actel.R [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                            dplyr::filter(Receiver != 'Release')
+                      
+# We don't have any environmental data in our detection extract here, but Actel 
+# will also find and plot temperature or other sensor values if you have those
+# kinds of tags.
+
+# Actel Receivers-------------------
+
+# Prepare and style entries for receivers
+actel_receivers <- full_receiver_meta %>% dplyr::mutate( Station.name = station,
```
Comment (naomitress) [inline code/day2/4_act_data_in_actel.R:134]: got a warning from running this part
```
Warning message:
Problem with `mutate()` input `Receiver`.
i NAs introduced by coercion
i Input `Receiver` is `as.integer(receiver)`. 
```

File: code/day2/5_actel_custom_spatial.R [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+# Let's design a spatial.txt file for our detection data
+
+# Let's first visualize our study area, with a popup that tells us what 
+# project each deployment belongs to:
+
+# Designing a spatial.txt file -----
+
+library(mapview)
+library(spdplyr)
```
Comment (naomitress) [inline code/day2/5_actel_custom_spatial.R:11]: had to run `install.packages("spdplyr")` should check setup.md to ensure its there

File: code/day2/5_actel_custom_spatial.R [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+rcvr_spatial <- our_receivers %>%    
+  dplyr::select(Longitude, Latitude) %>%           # and get a SpatialPoints object to pass to mapview
+  sp::SpatialPoints(CRS('+proj=longlat'))
+# and plot it using mapview. The popupTable() function lets us customize our tooltip
+mapview(rcvr_spatial, popup = popupTable(our_receivers, 
+                                         zcol = c("Array",
+                                                  "Station.name")))  # and make a tooltip we can explore
+
+# Can we design a spatial.txt file that fits our study area using 'glatos_array'
```
Comment (naomitress) [inline code/day2/5_actel_custom_spatial.R:33]: need to change `glatos_array` to act variable

**Review by naomitress — COMMENTED** [review]
4_.R looks fine, i got 1 warning no stops when i ran it
5_.R seem like a WIP

File: code/day2/5_actel_custom_spatial.R (hunk 2 above)
Comment (naomitress) [reply code/day2/5_actel_custom_spatial.R:11]: this is in setup.R i just seemed to have missed it

File: code/day2/5_actel_custom_spatial.R (hunk 3 above)
Comment (jdpye) [reply code/day2/5_actel_custom_spatial.R:33]: 5_.R is a WIP and is probably getting cut, mapview's current version is not working (!) and when i trield creating a spatial.txt file for ACT i hit another Actel error and i don't want to have to unravel that in the next 40 hours, and we have too much content for this timeline, so i'm thinking i leave it out.

Comment (jdpye) [reply code/day2/5_actel_custom_spatial.R]: Committing the WIP so we don't lose the work, but will remove it from the lesson markdown.

File: code/day2/4_act_data_in_actel.R (hunk 1 above)
Comment (jdpye) [reply code/day2/4_act_data_in_actel.R:138]: fixed this by reversing the filter and mutate steps to avoid trying to filter 'release' detections.

**Comment by jdpye** [comment]
@SoftwareMonk we good to merge this?

**Comment by jdpye** [comment]
i'm gonna merge it to hopefully keep people from having to clone the repo two days in a row

**Pull request merged**